src/main/java/org/sharedid/endpoint/model/AuditCookie.java

```diff
@@ -0,0 +1,399 @@
+package org.sharedid.endpoint.model;
```

Comment:
what domain is the audit cookie written to? & will the audit cookie function if its written in the sharedid.org domain?

Reply:
also, can this application geo detect anything?  If not how should we handle inits to sync?

Reply:
it's saved to sharedid.org. it's an analog of the one we use at Magnite on the Rubicon domain.

yes, it geo detects using a local database.

Reply:
got it so this functionality will work regardless of whether the entity deploying the code uses net acuity?

Reply:
that's correct. it uses an alternative to net acuity called GeoIP.